lnrpc/walletrpc/walletkit_server.go

```diff
@@ -186,6 +186,35 @@ var (
 		0x82, 0x54, 0x5d, 0x4b, 0xb4, 0xfa, 0xe0, 0x8b,
 		0xd5, 0x93, 0x78, 0x31, 0xb7, 0xe3, 0x8f, 0x98,
 	}
+
```

Comment:
nit: the commit message could be refined, it also introduces the allWitnessType ...

Reply:
yeah, I squashed the two commits and forgot to update the commit message. It now indicates that I added the map, and that I refactored the code to use the map.